### PR TITLE
Add script to ensure new PRs have docs

### PR DESCRIPTION
**Pull request by iankronquist**
While working on GWM I keep finding outdated or incomplete docs. I spoke with
@jordane on how to fix this and joked about making travis check that changes
were made to the docs. He recommended that I actually do that.

### CODE REVIEW

**Comment by tschuy** [comment]
Add docs about having this.

**Comment by iankronquist** [comment]
@tschuy the script works as intended. r?

File: scripts/travis-check-docs.bash [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+# Hello future developers! You are probably wondering whose stupid idea it was
+# to write this script, you just want to get your tests to pass and your code
+# to merge. An excellent sentiment! However, consider the state of the project
+# as a whole. When this was written, the docs and tests of GWM were out of date
```
Comment (tschuy) [inline scripts/travis-check-docs.bash]: > mfw gwm

**Comment by tschuy** [comment]
I'm just not sure we really need this at this point in the project. A lot of PRs will be implementing features that are already in the docs.

@Kennric thoughts?

**Comment by iankronquist** [comment]
rebased

**Comment by tschuy** [comment]
@Kennric would like to get this either merged or closed. I'm personally not in favor -- we have a lot of pull requests like "add checkActivities helper" that don't need any docs (comments should be more than enough for something like it).

**Comment by Kennric** [comment]
I think this is a bit too far, not every pr can reasonably include docs changes, especially since our user/api docs are in a different repo. Closing.

**Pull request closed without merging**